components/Badge/badge.css

```diff
@@ -0,0 +1,62 @@
+.badge-avatar {
+  position: relative;
+  width: fit-content;
+}
+.badge-avatar img {
```

Comment:
For readability, one empty line is suggested at the end of each code block

Reply:
Thanks Ruchi for pointing out. I'll keep that in mind.